model/vowpalWabbit/dffml_model_vowpalWabbit/vw_base.py

```diff
@@ -76,7 +69,7 @@ class VWConfig:
         "Feature to be used as `tag` in conversion of data to vowpal wabbit input format",
         default=None,
     )
-    convert_to_vw: bool = field(
+    requires_formatting: bool = field(
```

Comment:
I'm sorry, but we've gone back to the original behavior here.

From the [meeting minutes](https://docs.google.com/document/d/16u9Tev3O0CcUDe2nfikHmrO3Xnd4ASJ45myFgQLpvzM/edit#heading=h.hmuqg6k5xmp4):

> Because we want to be able to loop through all models with minimal tweaks to configs, let’s change conver_to_vw() to something a flag that indicates feature data is already in vw format, so that users opt-out of covert_to_vw() rather than opt-in.

For example

```python
    noconvert: bool = field(
        "Do not convert record features to vowpal wabbit input format",
        default=False,
    )